internal/cli/atlas/backup/exports/buckets/create.go

```diff
@@ -74,7 +74,7 @@ func CreateBuilder() *cobra.Command {
 		Use:   "create <bucketName>",
 		Short: "Create an export destination for Atlas backups using an existing AWS S3 bucket.",
 		Example: fmt.Sprintf(`  The following command creates an export destination for Atlas backups using the existing AWS S3 bucket named test-bucket:
-  $ %s backup export buckets create test-bucket --cloudProvider AWS --iamRoleID 12345678f901a234dbdb00ca`, config.BinName()),
+  $ %s backup export buckets create test-bucket --cloudProvider AWS --iamRoleId 12345678f901a234dbdb00ca`, config.BinName()),
```

Comment:
update docs?

Reply:
yes, my precommit hook is acting up and I disabled and forgot about it